Voycar.Api.Web/Features/Members/Post/Registration/Mapper.cs

```diff
@@ -0,0 +1,26 @@
+namespace Voycar.Api.Web.Features.Members.Post.Registration;
+
+using System.Security.Cryptography;
+using Entities;
+
+public class Mapper : Mapper<Request, Response, Member>
+{
+
+    public override Member ToEntity(Request r) => new()
+    {
+        Email = r.Email,
+        PasswordHash = BCrypt.Net.BCrypt.HashPassword(r.Password),
+        FirstName = r.FirstName,
+        LastName = r.LastName,
+        Street = r.Street,
+        HouseNumber = r.HouseNumber,
+        PostalCode = r.PostalCode,
+        Place = r.Place,
+        BirthDate = r.BirthDate,
+        BirthPlace = r.BirthPlace,
+        PhoneNumber = r.PhoneNumber,
+        IdCardNumber = r.IdCardNumber,
+        DriversLicenseNumber = r.DriversLicenseNumber,
```

Comment:
Ich würde gerne die Reihenfolge, wie bei den anderen Klassen einhalten. Siehe `Request` und `Member`.
```suggestion
        DriversLicenseNumber = r.DriversLicenseNumber,
        IdCardNumber = r.IdCardNumber,
```